task/task_smoke_test.go

```diff
@@ -1,9 +1,10 @@
 //go:build smoke
 
-package task
+package task_test
```

Comment:
Thanks! Black-box testing is better at this level. So, we make sure there is no dependency on private API.